src/Evolution/Systems/GrMhd/ValenciaDivClean/Subcell/GrTagsForHydro.hpp

```diff
@@ -0,0 +1,147 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <cstddef>
+#include <memory>
+#include <string>
+#include <unordered_map>
+#include <utility>
+
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/CoordinateMaps/CoordinateMap.hpp"
+#include "Domain/CoordinateMaps/Tags.hpp"
+#include "Domain/ElementMap.hpp"
+#include "Domain/FunctionsOfTime/FunctionOfTime.hpp"
+#include "Domain/FunctionsOfTime/Tags.hpp"
+#include "Domain/Tags.hpp"
+#include "Evolution/DgSubcell/Tags/Coordinates.hpp"
+#include "Evolution/DgSubcell/Tags/Inactive.hpp"
+#include "Evolution/DgSubcell/Tags/Mesh.hpp"
+#include "Evolution/DgSubcell/Tags/OnSubcellFaces.hpp"
+#include "Evolution/Initialization/InitialData.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+namespace Initialization {
+namespace Tags {
+struct InitialTime;
+}  // namespace Tags
+}  // namespace Initialization
+namespace Tags {
+struct AnalyticSolutionOrData;
+}  // namespace Tags
+namespace domain {
+namespace Tags {
+template <size_t Dim, typename Frame>
+struct Coordinates;
+template <size_t VolumeDim>
+struct Mesh;
+}  // namespace Tags
+}  // namespace domain
+// IWYU pragma: no_forward_declare db::DataBox
+/// \endcond
+
+namespace Initialization::subcell {
```

Comment:
neither of these two namespaces is showing up in my documentation

Reply:
The global `Initialization` namespaces has existed for a while, and I don't know where to document that. I can add documentation for the nested `subcell` namespace in this file. I'm not really sure where to put this mutator in general since it can work for multiple hydro systems but testing outside of a specific system would mean implementing an evolution system in the tests just to test the tags are set correctly, which seems like a lot of work for minimal or no gain.

Reply:
I guess the top-level namespaces could be documented in `docs/Namespaces.hpp` just like `docs/GroupDefs.hpp` (nothing needs to include these, they are just for doxygen).   We could also put other generic namespaces declarations there too.  I agree testing outside a  specific system makes no sense

Reply:
Yea, that seems reasonable. Do you want me to add a commit in this PR for the `docs/Namespaces.hpp` file?